src/rpc/claimtrie.cpp

```diff
@@ -142,11 +143,15 @@ UniValue getclaimsintrie(const UniValue& params, bool fHelp)
                 std::string sValue(vvchParams[1].begin(), vvchParams[1].end());
                 claim.push_back(Pair("value", sValue));
             }
+            std::string targetName;
+            CClaimValue targetClaim;
+            if (pclaimTrie->getClaimById(itClaims->claimId, targetName, targetClaim))
+                claim.push_back(Pair("name", targetName));
             claims.push_back(claim);
         }
 
         UniValue node(UniValue::VOBJ);
-        node.push_back(Pair("name", it->first));
+        node.push_back(Pair("normalizedName", it->first));
```

Comment:
This needs to be 'normalized_name'. After looking through the lbry codebase today, I saw how much they really want PEP8 names.